pydicom/examples/write_new.py

```diff
@@ -56,6 +56,12 @@
     ds.is_little_endian = True
     ds.is_implicit_VR = True
 
+    # Set creation date/time
+    dt = datetime.datetime.now()
+    ds.ContentDate = dt.strftime('%Y%m%d')
+    timeStr = dt.strftime('%H%M%S.%f') if microSeconds else dt.strftime('%H%M%S')
```

Comment:
Oh, right.  I had it as optional argument in my original function I copy-pasted it from.  Another option could be whether to add an UTC offset, but my main motivation was to offer something people could take as an example for how to format date / time.